### PR TITLE
Created the BlueSimple responsive nagivation

### PR DESCRIPTION
**Pull request by mehrab-alam**
BlueSimple-laptop
BlueSimple-mobile

### CODE REVIEW

File: marketing/src/index.js [hunk 1]
```diff
@@ -2,10 +2,13 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
+import BlueSimple from "./sections/navigation/BlueSimple/BlueSimple";
 
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
-root.render(<React.StrictMode></React.StrictMode>);
+root.render(<React.StrictMode>
+  <BlueSimple />
```
Comment (amezng) [inline marketing/src/index.js:10]: Don't change the index.js ever

File: marketing/src/sections/navigation/BlueSimple/BlueSimple.css [hunk 2]
```diff
@@ -1,42 +1,67 @@
...
 
-.features {
+.features .features-title {
+    padding-bottom: 5px;
+    transform-origin: left center;
+    transition: .1s;
+
+}
+
+.features .features-menu-inactive {
```
Comment (amezng) [inline marketing/src/sections/navigation/BlueSimple/BlueSimple.css:34]: Use context classes

**Review by amezng — CHANGES_REQUESTED** [review]

File: marketing/src/index.js (hunk 1 above)
Comment (mehrab-alam) [reply marketing/src/index.js]: removed all changes from index.js

File: marketing/src/sections/navigation/BlueSimple/BlueSimple.css (hunk 2 above)
Comment (mehrab-alam) [reply marketing/src/sections/navigation/BlueSimple/BlueSimple.css]: used context classes

File: marketing/src/index.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
+import BlueSimple from "./sections/navigation/BlueSimple/BlueSimple";
```
Comment (amezng) [inline marketing/src/index.js:5]: remove this import

**Review by amezng — CHANGES_REQUESTED** [review]

**Review by amezng — APPROVED** [review]

**Pull request merged**